docs/find_and_replace.md

```diff
@@ -91,6 +91,12 @@ After reviewing the 'Important Notes & Workflow Recommendations' below, please s
 
     The Actions available through Archipelago's Advanced Batch Find and Replace can potentially have repository-wide effects. It is strongly recommended that you proceed with caution when executing any of the available Actions. 
 
+
+!!! warning "Adding New Facets"
+
+    The default Facets available through Archipelago's Advanced Batch Find and Replace have an important configuration selection made on each individual Facet. For every [new Facet you add](strawberry_key_name_providers.md) for Find and Replace, you need to select the checkboxes for both the 'VBO batch handler' settings to use the `VBO Batch Facet processor`, and the selection within the 'VBO batch handler settings' to `Use URL based facets in VBO Batches`. You need to make sure these are selected so that the "visible" list/count of objects you filter using a Facet is respected during actual VBO process execution of batch changes you make for any Find and Replace Actions.
```

Comment:
Ok! Here is a missing need/documentation part that needs to be added. VBO does not pass a "limit" (except if your VIEW has actually a "SHOW" a defined number of results which most users will never use. Because of that, when run will default to the Search API/Solr defined Limit. At 
`http://localhost:8001/admin/config/search/search-api/server/esmero_solr/edit` (under the advanced tab). Which means if you see 1000 objects In your Find and Replace and select all and run IT and that value here is 100, only 100 will be processed. Nothing we can do around that (for now, except open an ISSUE, tag me, maybe we can find a way). We need a note for that 🐝 !

Reply:
Ok, added an additional statement about this behavior, using some of the language you shared in your above comment. Let me know what think @DiegoPino